test/nostrum/shard/dispatch_test.exs

```diff
@@ -0,0 +1,22 @@
+defmodule Nostrum.Shard.DispatchTest do
+  use ExUnit.Case, async: true
+
+  alias Nostrum.Shard.Dispatch
+  alias Nostrum.Struct.Event.MessageDelete
+
+  describe "handle_event/1" do
+    test "returns `%MessageDelete{}` given Message Delete Event" do
```

Comment:
I don't think there's much sense in trying to format the struct here in the code markdown, we could just have it reference the message delete struct with plain good ol' fashion english.

Reply:
Resolved in e3a0ce7.